WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -58,8 +58,9 @@ final class PrefixAllGlobalsSniff extends AbstractFunctionParameterSniff {
 	/**
 	 * Minimal number of characters the prefix needs in order to be valid.
 	 *
-	 * @since n.e.x.t.
+	 * @since 2.2.0
```

Comment:
Still missing a second tag `@since 3.2.0 The minimum number of characters was raised from 3 to 4`.

Reply:
ok!

Reply:
@davidperezgar You've now removed the original `@since` tag (again...) and still not added the extra `@since` tag with explanation of what's changed.
The extra test needed is also still missing.

If something is unclear: read documentation, check existing code for how it's done and ask questions. In the mean time, this is now the third time I've reviewed and this tiny PR is still not mergable.

Reply:
I didn't know that it has to be both. I update the tag.